public/js/addTask.js

```diff
@@ -7,24 +7,25 @@ async function taskFormHandler(event) {
       window.location.toString().split('/').length - 1
```

Comment:
I wrote a comment to Daniel about using a data selector to pass this information.  Maybe more to come?

Reply:
I was hoping you or Daniel could assist with this. I will leave it for now.